spec/lib/api/v3/work_packages/work_package_representer_spec.rb

```diff
@@ -493,7 +483,15 @@
 
       context 'when the user has the permission to add and remove watchers' do
         it 'should have a link to add watcher' do
-          expect(subject).to have_json_path('_links/addWatcher/href')
+          expect(subject).to be_json_eql(
+                               api_v3_paths.work_package_watchers(work_package.id).to_json)
+            .at_path('_links/addWatcher/href')
+        end
+
+        it 'should have a link to remove watcher' do
+          expect(subject).to be_json_eql(
+                               api_v3_paths.watcher('{user_id}', work_package.id).to_json)
+                               .at_path('_links/removeWatcher/href')
```

Comment:
This indentation looks weird 0o. I'd expected these lines to be either more to the right, which might not be possible due to the line limit, or to the left (indented only once).

But since @houndci didn't have to say anything about this I'm a little confused and this is not important at all.